cmd/snapshot-create.go

```diff
@@ -0,0 +1,161 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"time"
+
+	"github.com/Layr-Labs/sidecar/internal/logger"
+	"github.com/spf13/cobra"
+)
+
+var (
+	snapshotCreateToPath string
+	snapshotFidelity     string
```

Comment:
Rename to `snapshotType` and make it an enum to show the different types available.